packages/spacecat-shared-data-access/test/unit/service/import-url/index.test.js

```diff
@@ -60,7 +60,7 @@ describe('Import Url Tests', () => {
       it('should return null when an item is not found', async () => {
         mockDynamoClient.getItem.resolves(null);
         const result = await exportedFunctions.getImportUrlById('test-id');
-        expect(result).to.be.null;
+        expect(result).to.equal(null);
```

Comment:
Are these not equivalent? I actually prefer the readability of the original `.to.be.null`.

Reply:
The eslint check is failing - so, I had to modify the unit test.

Reply:
I checked out the branch and tried `npm run lint -- --fix` and it did not complain for me. Can we leave it as `.to.be.null`? That seems like the current standard, used 35x in the codebase.